app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Admin
+  class UsersController < AdminController
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+
+  # GET /users
+  # GET /users.json
+  def index
+    @users = User.all
```

Comment:
@rishabhguptarishi Lets display paginated list of users in admin. for pagination use https://github.com/kaminari/kaminari